src/python/director/drakevisualizer.py

```diff
@@ -200,6 +200,49 @@ def __init__(self, name, geom, polyData, parentTransform):
             self.polyDataItem.shadowOn()
 
 
+class LidarSource(object):
+
+    def __init__(self, view):
+        self.lidars = {}
+        self.view = view
+
+        lcm = lcmUtils.getGlobalLCMThread()
+        self.lidarAggregator = PythonQt.dd.ddLidarPointCloudLCM(lcm)
+
+        self.lidarAggregator.channelAdded.connect(self.addChannel)
+
+        self.channels = set(self.lidarAggregator.channels())
+        self.lidarAggregator.init(lcm)
+
+        self.timer = TimerCallback(targetFps=30)
+        self.timer.callback = self._updateSource
+
+    def start(self):
+        self.timer.start()
+
+    def stop(self):
+        self.timer.stop()
+
+    def addChannel(self, channel):
+        self.channels.add(channel)
+
+    def _updateSource(self):
+        for channel in self.channels:
+            pd = vtk.vtkPolyData()
+            self.lidarAggregator.extractPolyData(channel, pd)
+
+            if not channel in self.lidars:
+                name = '%s lidar source' % channel
+                pdObj = vis.PolyDataItem(name, pd, self.view)
```

Comment:
you should add a line like:

        om.addToObjectModel(pdObj, parentObj=om.getOrCreateContainer('lidar sensors'))

This will create an entry in the object model for this lidar source.  Other geometry in the scene is organized by link and by robot.  In the ongoing drake lcm type discussion, we might consider how we can associate the lidar sensor with a particular robot in the scene, if needed.